hudi-spark-datasource/hudi-spark3-common/src/test/java/org/apache/hudi/spark3/internal/TestHoodieDataSourceInternalBatchWrite.java

```diff
@@ -70,7 +71,7 @@ private static Stream<Arguments> bulkInsertTypeParams() {
   @ParameterizedTest
   @MethodSource("bulkInsertTypeParams")
   public void testDataSourceWriter(boolean populateMetaFields) throws Exception {
-    testDataSourceWriterInternal(Collections.EMPTY_MAP, Collections.EMPTY_MAP, populateMetaFields);
+    testDataSourceWriterInternal(Collections.emptyMap(), Collections.emptyMap(), populateMetaFields);
```

Comment:
just asking, does it make any difference? I believe both are the same thing right?